gateway_to_backend/protocol_buffers_files/error.proto

```diff
@@ -43,4 +43,8 @@ enum ErrorCode {
     INVALID_MAX_HOP_COUNT = 25;
     SINK_OUT_OF_MEMORY = 26;
     SINK_TIMEOUT = 27;
+    INVALID_SCRATCHPAD_CHUNK_ID = 28;
+    // If scratchpad chunk size is bigger than
+    // the max value supported by the gateway
+    INVALID_SCRATCHPAD_CHUNK_SIZE = 29;
```

Comment:
Given that we don't allow out of order chunks, should there be a separate error code for indicating that backend is trying to upload a chunk that is out of sync (e.g. `INVALID_SCRACTCHPAD_CHUNK_START_OFFSET` or something like that).